### DISCUSSION TITLE
How to disable evilified-state completely?

### DISCUSSION THREAD

**Discussion by pcompassion** [syl20bnr/spacemacs#16128] [Q&A]
I can't seem to turn off `evilified-state` comming up 
(maybe it comes up when I press `C-z` , after sometime If open some new buffer then the new buffer might get `evilified`?

```
(defun dotspacemacs/layers ()


     (spacemacs-editing :variables
              hybrid-style-enable-evilified-state nil
              hybrid-style-enable-hjkl-bindings nil
              hybrid-style-use-evil-search-module nil
              hybrid-style-default-state 'emacs
              )


```

**Comment by Nikratio** [comment]
Have you tried setting `evil-buffer-regexps` to something that matches every mode? https://www.spacemacs.org/doc/FAQ.html#disable-evilification-of-a-mode
